packages/@react-spectrum/form/docs/Form.mdx

```diff
@@ -0,0 +1,224 @@
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/form';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+ import {Checkbox} from '@react-spectrum/checkbox';
+ import {Form} from '@react-spectrum/form';
+ import {Radio, RadioGroup} from '@react-spectrum/radio';
+ import {TextField} from '@react-spectrum/textfield';
+ ```
+
+# Form
+
+<p>{docs.exports.Form.description}</p>
+
+<HeaderInfo
+ packageData={packageData}
+ componentNames={['Form']} />
+
+## Example
+
+```tsx example
+<h3 id="label-1">Login</h3>
+<Form width={400} aria-labelledby="label-1">
+  <TextField label="Email" placeholder="abc@adobe.com" />
+  <TextField label="Password" placeholder="1234" />
+  <Checkbox>Remember me</Checkbox>
+</Form>
+```
+
+## Content
+
+Form accepts one or more children to render as its form elements. Label alignment, positioning, and other [properties](#visual-options)
+are set on the Form itself and propagated to its children to generate a cohesive design. To override these top level Form props on individual children, 
+set them directly on the child instead.
+
+```tsx example
+<h3 id="label-2">Sign Up</h3>
+<Form width={400} aria-labelledby="label-2" isRequired necessityIndicator="label">
+  <TextField label="Name" placeholder="Bob Doe" />
+  <TextField label="Email" placeholder="abc@adobe.com" />
+  <TextField label="Address" placeholder="123 Any Street" isRequired={false} />
+</Form>
+```
+
+### Accessibility
+
+As per [W3 Form guidelines](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/form.html), the controls provided to the form should 
+use native host semantics (`button`, `input`, `select`, `textarea`) whenever possible.
+
+## Labeling
+
+### Accessibility
+
+If a label is provided to the Form by adding either the `aria-label` or `aria-labelledby` prop, 
+the element will be identified to assistive technology as a form landmark region. 
+
+```tsx example
+<h3 id="label-3">Personal Information</h3>
+<Form width={400} aria-labelledby="label-3">
+  <TextField label="First Name" placeholder="Bob" />
+  <TextField label="Last Name" placeholder="Dole" />
+  <RadioGroup label="Favorite pet" name="favorite-pet-group">
+    <Radio value="dogs">Dogs</Radio>
+    <Radio value="cats">Cats</Radio>
+    <Radio value="dragons">Dragons</Radio>
+  </RadioGroup>
+</Form>
+```
+
+
+## Props
+
+<PropTable component={docs.exports.Form} links={docs.links} />
+
+## Visual Options
+
+### Label Position
+
+```tsx example
+<Form width={400} marginBottom="20px" labelPosition="top">
```

Comment:
The prop table has the default and the possible values. It's nice to see both states shown together. I'm trying to figure out if it is better to just have one example (bare minimum), what you have, or go for completeness and explicitly state the default and all possible values? Does the tech writing class you taking have a suggestion on this? Like are people able to look around the docs and figure this out, or do we need to repeat things as often as possible so they get it?

Reply:
So the tech writing class definitely emphasizes making things concise. That being said, people typically gravitate to visuals so I feel like the visual examples should show as much as possible but within reason. 

IMO for props that only have one/two (3 probably is stretching it) options we can shoot for visual completeness, but anything more we can go with a single example and link to storybook/spectrum/other documentation